pkg/skaffold/deploy/kubectl.go

```diff
@@ -35,6 +35,8 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var warner Warner = &logrusWarner{}
```

Comment:
Maybe add a comment explaining that this is only used so that we can test warnings?